abci/client/socket_client.go

```diff
@@ -417,6 +427,23 @@ func (cli *socketClient) ApplySnapshotChunkSync(
 	return reqres.Response.GetApplySnapshotChunk(), cli.Error()
 }
 
+func (cli *socketClient) PrepareProposalSync(
+	req types.RequestPrepareProposal,
+) (*types.ResponsePrepareProposal, error) {
+
+	reqres := cli.queueRequest(types.ToRequestPrepareProposal(req))
+	return reqres.Response.GetPrepareProposal(), nil
+}
+
+func (cli *socketClient) ProcessProposalSync(
+	req types.RequestProcessProposal,
+) (*types.ResponseProcessProposal, error) {
+
+	reqres := cli.queueRequest(types.ToRequestProcessProposal(req))
+
+	return reqres.Response.GetProcessProposal(), nil
```

Comment:
Comment for `PrepareProposalSync` also applies to `ProcessProposalSync`

Reply:
Resolved by https://github.com/celestiaorg/celestia-core/pull/902/commits/3c0dd40d84ba5608b11f3f3c617d4c5c90b12473